common/js/src/popup-window/in-popup-main-script.js

```diff
@@ -64,8 +64,8 @@ GSC.PopupWindow.Client.showWindow = function() {
  * window.
  * @param {*} result
  */
-GSC.PopupWindow.Client.resolveModalDialog = function(result) {
-  const callback = GSC.PopupWindow.Client.getData()['resolveModalDialog'];
+GSC.PopupWindow.InPopupMainScript.resolveModalDialog = function(result) {
+  const callback = GSC.PopupWindow.InPopupMainScript.getData()['resolveModalDialog'];
```

Comment:
nit: This line is longer than 80 characters. Please use /format-code.sh to let clang-tidy reflow this line.